src/SFA.DAS.FAA.Web/Controllers/Apply/DisabilityConfidentController.cs

```diff
@@ -73,12 +73,12 @@ public async Task<IActionResult> Post(Guid applicationId, DisabilityConfidentVie
             });
 
             return RedirectToRoute(RouteNames.ApplyApprenticeship.DisabilityConfidentConfirmation,
-                new { ApplicationId = applicationId });
+                new { ApplicationId = applicationId, isEdit });
         }
 
         [HttpGet]
         [Route("apply/{applicationId}/disability-confident/confirm", Name = RouteNames.ApplyApprenticeship.DisabilityConfidentConfirmation)]
-        public async Task<IActionResult> GetSummary([FromRoute] Guid applicationId)
+        public async Task<IActionResult> GetSummary([FromRoute] Guid applicationId, [FromQuery] bool isEdit = false)
         {
             var result = await mediator.Send(new GetDisabilityConfidentQuery
```

Comment:
This endpoint still seems to be sharing the same query, and therefore the same outer api endpoint, as the initial data entry page.

Reply:
Based on the Edit value only we could derive the BackLinkUrl, also it helps to differentiate the user came from task list page or edit screen. so i believe this is required